### ISSUE TITLE
Version is now Unstable 

### ISSUE THREAD

**Issue by DesperationTeddy** [Slivo-fr/elinor-reloaded#24]
this program is by far the single best program I have ever used, In fact I became reliant on it because there simply is no alternative out there, since it became unstable I have tried everything in my limited knowledge to get it back up and running, uninstalled and re installed, even uninstalled EVE and reinstalled that. even considering a complete window reset to see if that will solve it. however I believe, with my very limited computer knowledge that its to do with the recent windows .NET Frame update. tried running in compatibility trouble-shooter to no avail.  From what I have been able to ascertain the player that created this no longer plays and so this is a shot in the dark but I have to try. in the hope someone has the skills to get it up and running.

**Comment by Slivo-fr** [comment]
Hi,

Thanks for reporting your problem.

Could you please describe what you mean by unstable?
Can you provide any error message?
Could you share a bit more regarding your own setup? (is version,. Net version, anything relevant)

**Comment by DesperationTeddy** [comment]
Yeah I'm currently at work for the next couple of days. I'll sit down and
write what the problem is with it however I ain't that computer savvy.

Sometimes when I load it up it closes immediately without an error message
and sometimes it'll run but after about say 5 or 6 exports it lags and
shuts down.

I can get my system information and what I have tried thus far to resolve
it.

Beyond that my technical knowledge is limited hence why I have sought your
help

Regards
           Fish

**Comment by Slivo-fr** [comment]
Thanks for the details

It feels weird to me that such a crash only has one user reporting it 🤔 
Could you please share your OS version and .Net version ?

Any chance you can record a short video of the crash ?

**Comment by DesperationTeddy** [comment]
Yeah slivo will do mate will be around 21:00 eve time. Just at work at the
minute I didn't forgot just been doing alot of hours coz of inflation and
bills. I'll record and sent information and drop it in a file dump and link
it for you mate later on this evening

**Comment by brubsby** [comment]
Elinor reloaded is also crashing for me pretty regularly. Version 1.12.0

To reproduce: 
Run program
Load profile (buy range 1 jump, sell range 1 jump), Corp standing 4.03, Faction standing 2.57, skills 5.
Pin
Enable auto copy
Export item (200mm Crystalline Carbonide Restrained Plates)


Expected: Sell price is copied (186500)
Actual: Buy Price is copied (99310)


Continuing:
Toggle Auto copy field to Buy Price
Export market data to file
Autocopy symbol turns into yellow circle
Program crashes
Buy Price still on clipboard (unclear if from before)

Checking the install directory, `C:\Program Files (x86)\Elinor reloaded\` doesn't reveal any crash logs, and there doesn't seem to be any options to debug in the UI, so I'm not sure if there's any other help I can provide

**Comment by Slivo-fr** [comment]
Good afternoon
Would you have any other program that could interact with the clipboard ?
What is your .net version

**Comment by brubsby** [comment]
Unclear which installation of .net Elinor is running on. Using evemon or jeveassets has no issue copying from and reading clipboard, though I'm unsure if either use .net to interact with the clipboard.

```
>dotnet --list-runtimes
Microsoft.NETCore.App 6.0.36 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.NETCore.App 8.0.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
Microsoft.WindowsDesktop.App 6.0.36 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
Microsoft.WindowsDesktop.App 8.0.16 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

Also, it does successfully work for buy orders, but not sell orders, so my guess is that the error has something more to do with parsing the output files than interfacing with the clipboard. Perhaps a game update slightly altered the export file formats or something.

**Comment by Slivo-fr** [comment]
Could you please provide a market export file so I can try reproduce the issue ?

**Comment by brubsby** [comment]
[The Forge-Pith X-Type Explosive Shield Hardener-2025.05.31 221906.txt](https://github.com/user-attachments/files/20536280/The.Forge-Pith.X-Type.Explosive.Shield.Hardener-2025.05.31.221906.txt)

This file, or this file:

[The Forge-Pith X-Type Explosive Shield Hardener-2025.05.31 221909.txt](https://github.com/user-attachments/files/20536281/The.Forge-Pith.X-Type.Explosive.Shield.Hardener-2025.05.31.221909.txt)

Caused a crash when copying buy prices, though it was working for a bit beforehand.

**Comment by Slivo-fr** [comment]
Thank you
I will try to reproduce the issue on my end

**Comment by Slivo-fr** [comment]
May I ask how many files do you have in the market export folder out of curiosity ?

**Comment by brubsby** [comment]
I can reproduce the issue with any number of files, and I often click the bottom right of the app to delete them.

**Comment by Slivo-fr** [comment]
Could you please send me a message on discord (user `slivo`), I would like to get the list of all program you have installed.
At this point I feel like there is something also interacting with your clipboard leading to your crashes
